libs/ui-toolkit/src/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,73 @@
+import classnames from 'classnames';
+import type { ChangeEvent } from 'react';
+
+// Supports controlled and uncontrolled setups.
+interface ToggleProps {
+  label: string;
+  value: string;
+}
+
+export type TogglesArr = [ToggleProps, ToggleProps, ...ToggleProps[]];
```

Comment:
```suggestion
```

Reply:
This ensures it expects a minimum of two args